### PR TITLE
[CARBONDATA-1843] Block CTAS and external table feature

### PR DESCRIPTION
**Pull request by jackylk**
1. Block 'external' syntax
2. Block 'CTAS' syntax

 - [X] Any interfaces changed?
 No
 - [X] Any backward compatibility impacted?
 No
 - [X] Document update required?
No
 - [X] Testing done
Added
 - [X] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
NA

### CODE REVIEW

File: integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala [hunk 1]
```diff
@@ -58,7 +59,12 @@ class CarbonSession(@transient val sc: SparkContext,
    */
   @transient
  override lazy val sharedState: SharedState = {
-    existingSharedState.getOrElse(new SharedState(sparkContext))
+    if (existingSharedState.isDefined) {
```
Comment (ravipesala) [inline integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: Better use `match {case}`

Comment (jackylk) [reply integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: ok, fixed

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1655/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/2043/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/2044/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/2045/

File: integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala [hunk 2]
```diff
@@ -170,8 +181,15 @@ object CarbonSession {
...
-        CommonUtil.cleanInProgressSegments(
-          carbonProperties.getProperty(CarbonCommonConstants.STORE_LOCATION), sparkContext)
+        try {
+          CommonUtil.cleanInProgressSegments(
+            carbonProperties.getProperty(CarbonCommonConstants.STORE_LOCATION), sparkContext)
+        } catch {
+          case e: Throwable =>
+            // catch all exceptions to avoid CarbonSession initialization failure
+          LogServiceFactory.getLogService(this.getClass.getCanonicalName)
+              .error(e, "Failed to clean in progress segments")
```
Comment (QiangCai) [inline integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala]: indentation is incorrect

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/2049/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1659/

**Pull request closed without merging**